thehive4py/endpoints/cortex.py

```diff
@@ -27,3 +27,26 @@ def create_responder_action(
                 "responderId": responder_id,
             },
         )
+
+    def list_analyzers(self) -> dict:
```

Comment:
So it looks like we miss a query parameter `range` based on the [api docs](https://docs.strangebee.com/thehive/api-docs/#tag/Cortex/operation/List%20Analyzers)

Additionally I'd also refine the output type hint as according to the docs it should be a list of analyzers.

For that please define the module `thehive4py/types/cortex.py` and add something like: 

```python
# thehive4py/types/cortex.py
from typing import List, TypedDict


class OutputAnalyzertRequired(TypedDict):
    id: str
    name: str
    version: str
    description: str


class OutputAnalyzer(OutputAnalyzerRequired, total=False):
    dataTypeList: List[str]
    cortexIds: List[str]
```


```suggestion
    def list_analyzers(self, range: str) -> List[OutputAnalyzer]:
```